src/utils/worker.js

```diff
@@ -11,27 +11,29 @@ let msgId = 1;
  * @memberof utils/utils
  * @static
  */
-function workerTask(worker: any, method: string) {
+function workerTask(retrieveWorker: Function, method: string) {
```

Comment:
I wasn't sure how Flow would type this function that takes nothing and returns a Promise.